features/cursor-style.yml.dist

```diff
@@ -0,0 +1,126 @@
+# Generated from: cursor-style.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2021-12-07
+  baseline_high_date: 2024-06-07
+  support:
+    chrome: ≤83
+    chrome_android: "83"
+    edge: "83"
+    firefox: ≤72
+    firefox_android: "95"
+    safari: ≤13.1
+    safari_ios: ≤13.4
```

Comment:
Across the board, caniuse says that Safari doesn't support this feature but BCD says it does. See https://github.com/Fyrd/caniuse/issues/7066. I've filed https://github.com/mdn/browser-compat-data/issues/24142 to see if that can/should be fixed. I think this PR is going to be blocked for a little bit until we've figured who's correct.

(I'll wait for triage on BCD to see if it's correctable soon—if not, we can turn this into a draft feature so the PR doesn't have to be held open for forever.)